web/src/utils/zod.ts

```diff
@@ -49,6 +49,15 @@ export const paginationZod = {
   ),
 };
 
+export const optionalPaginationZod = {
```

Comment:
note: could re-use in evals router in follow-up PR

Reply:
sounds good as quick improvement